run_pipeline.py

```diff
@@ -13,9 +13,19 @@
 from model.layers import get_linear_layers, get_SAGEConv_layers
 from single_epoch import test
 
+from reporting.wandb import setup_config, report_results
+from reporting.types import (
+    Stats,
+    BaseStats,
+    ContinousStatsVal,
+    ContinousStatsTest,
+    ContinousStatsTrain,
+)
 
-def run_pipeline(config: Config):
+
+def run_pipeline(config: Config, with_wandb: bool = False):
     config.print()
+    wandb, config = setup_config("Fashion-Recomm-GNN", with_wandb, config)
```

Comment:
do you mind if I ask this to be optional?
1. we don't want our integration test to post stuff to wandb
2. we don't want to wait for wandb to start up at development / debug time - it was really too slow for that!

thanks!

Reply:
I mean, sorry, should we add this to the Config file?